element_calcium_imaging/scan.py

```diff
@@ -22,6 +22,8 @@ def activate(
 ):
     """Activate this schema.
 
+    Required dependencies should be defined in the linking module
+
```

Comment:
```suggestion
```

Reply:
Sorry for the confusion.  #51 was already completed but the issue wasn't closed.  The fix is [here](https://github.com/datajoint/element-calcium-imaging/blob/605f91b97f2d3c594a0e827c7b0eb5d613ae9808/element_calcium_imaging/scan.py#L32-L41).

Reply:
Okay okay sir, No problem. I'm glad the issue was already fixed.